apps/labs/posts/czi-eoss-5-conda-forge.md

```diff
@@ -0,0 +1,285 @@
+---
+title: 'Two years of contributions to conda-forge: work done during our CZI EOSS 5 grant'
+published: January 20, 2025
```

Comment:
TODO: update when publishing

Reply:
```suggestion
published: February 10, 2025
```

Assuming we are aiming for a Monday merge.